docs/docs/tutorials/using_dask.md

```diff
@@ -204,25 +274,10 @@ This dataset is saved in parquet format, a column-oriented file format commonly
 
 Performing this kind of analysis on such a large dataset would not be possible without a tool like Dask. On Nebari, Dask comes out-of-the-box ready to help you handle these larger-than-memory (out-of-core) datasets.
 
-### The Dask diagnostic UI
-
-Dask comes with an inbuilt dashboard containing multiple plots and tables containing live information as
-the data gets processed. Let's understand the dashboard plots `Task Stream` and `Progress`.
-The colors and the interpretation would differ based on the computation we choose.
-
-Each of the computation in split into multiple tasks for parallel execution. From the progress bar we see 4
-distinct colors associated with different computation. Under task stream (a streaming plot) each row represents a thread
-and the small rectangles within are the individual tasks.
-
-Check out the [Dask Documentation](https://docs.dask.org/en/stable/dashboard.html) for more information.
-
-![Dask diagnostic UI - showing four parallel computation streams](/img/tutorials/dask_diagostic_UI.png)
-
-### Shutting down the cluster
+## Step 5 - Shutdown the cluster
 
 As will you have noticed, you can spin up a lot of compute really quickly using Dask.
 
-:::warning
 **With great power comes great responsibility**
 
 Remember to shut down your cluster once you are done, otherwise this will be running in the background, and you might incur on unplanned costs.
```

Comment:
```suggestion
Remember to shut down your cluster once you are done, otherwise this will be running in the background, and you might incur unplanned costs.
```